docs/docs/how_to_run_sim.md

```diff
@@ -0,0 +1,81 @@
+# Running the simulator
+
+_Note: This document is intended for the organizers of events in which the RCJ
+Soccer Sim is to be used_
+
+This document outlines how the RoboCupJunior Soccer Sim can be used to simulate
+a match in a "headless" way. In other words, it shows how you can go from
+having the code for two teams and the RoboCupJunior Soccer Sim code to getting
+an output in form of either an MPEG-4 video or a HTML site.
+
+**Input**
+- Source code for the yellow team
+- Source code for the blue team
+- RoboCupJunior Soccer Sim
+
+**Output**
+- MPEG-4 video or HTML site
+
+## Preliminaries
+
+This guide makes a couple assumptions:
+1. You use a UNIX-like environment (i.e. something like Linux or macOS)
```

Comment:
```suggestion
1. You use an UNIX-like environment (i.e. something like Linux or macOS)
```

Reply:
@Adman I guess I don't care about that that much :slightly_smiling_face: